worlds/AutoWorld.py

```diff
@@ -462,3 +483,17 @@ def data_package_checksum(data: "GamesPackage") -> str:
     assert sorted(data) == list(data), "Data not ordered"
     from NetUtils import encode
     return hashlib.sha1(encode(data).encode()).hexdigest()
+
+
+def _normalize_description(description):
```

Comment:
shouldn't this just be https://docs.python.org/3/library/inspect.html#inspect.cleandoc  ?